src/utils/_colors.scss

```diff
@@ -12,23 +12,47 @@
 // $c-primary-dark: color-variation($c-primary, -2) -> 2 steps darker
 // $c-primary-darkest: color-variation($c-primary, -4) -> 4 steps darker
 
+$color-variation-dark: black !default;
+$color-variation-light: white !default;
+
 $color-lighten-steps: 25, 50, 75, 90, 95 !default;
 $color-darken-steps: 20, 35, 55, 75 !default;
 
-@function color-variation($color, $step: 0, $dark: black, $light: white) {
+@function color-variation(
+  $color,
+  $step: 0,
+  $color-variation-dark,
+  $color-variation-light
+) {
```

Comment:
you need to set `$color-variation-dark` and `$color-variation-light` with global vars (`$color-variation-dark` and `$color-variation-light`, maybe this two with an other name)

Reply:
done